implementation/src/main/java/io/smallrye/mutiny/operators/AbstractUni.java

```diff
@@ -17,21 +17,23 @@
 
 public abstract class AbstractUni<T> implements Uni<T> {
 
-    protected abstract void subscribing(UniSubscriber<? super T> subscriber);
+    public abstract void subscribe(UniSubscriber<? super T> subscriber);
```

Comment:
Should we add a bit of javadoc?

Reply:
Yes since it's now part of the public API.